lib/include/jxl/cms_interface.h

```diff
@@ -0,0 +1,210 @@
+/* Copyright (c) the JPEG XL Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @file cms_interface.h
+ * @brief Interface to allow the injection of different color management systems
+ * (also called color management modules, or CMMs) in JPEG XL.
```

Comment:
A bit strange to introduce the acronym CMM here while everywhere else we're going with CMS.

Reply:
To me this looks good, and CMS is more common I think so I'm ok with CMS anywhere else outside of this mention

Reply:
Yes, introducing both acronyms and then using one of them looks fine, it was just weird to introduce CMM but not CMS when CMS is the one that gets used.